docs/meson.build

```diff
@@ -0,0 +1,19 @@
+custom_target(
+        'man_dbus_broker',
+        output : 'dbus-broker.1',
+        input : 'dbus-broker.rst',
+        command : [rst2man, '@INPUT@', '@OUTPUT@'],
+        install : true,
+        install_dir : join_paths(get_option('mandir'), 'man1')
```

Comment:
We don't put spaces before the colon ;)

Also, doesn't this need: `build_by_default: true`
Or is the `install: true` enough to pull this in?

Reply:
It appears to be enough.

Reply:
Fixed the whitespace.